src/sections/VotingSection.tsx

```diff
@@ -21,13 +22,9 @@ export const VotingSection = ({
   ballot,
   onVote,
 }: VotingSectionProps) => {
-  const [language, setLanguage] = useState<keyof TranslatableText>("en");
+  const { language } = useCurrentLanguage();
```

Comment:
idk maybe
```suggestion
  const { languageIdentifier } = useCurrentLanguage();
```
or with my `useCurrentLanguage()` suggestion above
```suggestion
  const languageIdentifier = useCurrentLanguage();
```